### PR TITLE
Update & Add New Commercial Platforms

### PR DESCRIPTION
**Pull request by zhimin-z**
This PR 
1) adds some well-known commercial platforms (=closed source platforms) for MLOps:
- ModelOp
- Domino
- Databricks (https://www.databricks.com/product/machine-learning)
2) Update link metadata
- IBM Watson
- Weights & Biases

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -195,6 +195,7 @@ This repository contains a curated list of awesome open source libraries that wi
 * [TorchServe](https://pytorch.org/serve/) ![](https://img.shields.io/github/stars/pytorch/serve.svg?style=social) - TorchServe is a flexible and easy to use tool for serving PyTorch models.
 * [Transformer-deploy](https://github.com/ELS-RD/transformer-deploy/) ![](https://img.shields.io/github/stars/ELS-RD/transformer-deploy.svg?style=social) - Transformer-deploy is an efficient, scalable and enterprise-grade CPU/GPU inference server for Hugging Face transformer models.
 * [Triton Inference Server](https://github.com/triton-inference-server/server) ![](https://img.shields.io/github/stars/triton-inference-server/server.svg?style=social) - Triton is a high performance open source serving software to deploy AI models from any framework on GPU & CPU while maximizing utilization.
+* [Weights & Biases](https://github.com/wandb/wandb) ![](https://img.shields.io/github/stars/wandb/wandb.svg?style=social) - Machine learning experiment tracking, dataset versioning, hyperparameter search, visualization, and collaboration.
```
Comment (axsaucedo) [inline README.md:198]: wandb is primarily a closed source platform so this will have to be moved to the lower section

**Review by axsaucedo — CHANGES_REQUESTED** [review]
Added a comment to move WnB but other than that looks great, thank you

**Comment by zhimin-z** [comment]
Thanks, Alejandro. I initially thought it is close-source, but I found I was missing something: here is the GitHub repo for wandb: https://github.com/wandb/wandb. They did open their source afterward. To keep the list consistent, I suggest moving wandb out of closed-sourced platforms. @axsaucedo

**Comment by axsaucedo** [comment]
Unfortunately wandb would not work without access to their closed source service, looking at the documentation you still need to enter a license to be able to use it so it's still quite an ambiguous one https://docs.wandb.ai/guides/self-hosted. It seems there's a license for a single user, but given it's closed it's still quite a strange one, for now I would keep it on the commercial products as this seems more of a "freemium" version compared to the other libraries which are actually available for full use.

**Comment by zhimin-z** [comment]
If we categorize software based on the existence of partial licensing rather than free and open-source to personal users, then it seems a number of tools should be reconsidered in the module of other parts. For example, [Pachyderm](https://www.pachyderm.com/) is [open-source](https://github.com/pachyderm/pachyderm)  but it is [source-available limited license](https://github.com/pachyderm/pachyderm/blob/master/LICENSE). I believe there any other tools for similar situations tool. We need to check them one by one.

**Comment by zhimin-z** [comment]
If you check the licensing of [W&B](https://github.com/wandb/wandb/blob/main/LICENSE), it is MIT licensed. Permission is hereby granted free of charge to any person obtaining a copy of this software and associated documentation files. That is the main reason why I propose to rethink what licensing criteria shall we categorize the tools. If the tool is source-available limited licensed, shall we put it into commercial platforms?

**Comment by axsaucedo** [comment]
I do see your point in regards to projects like pachyderm which may have source-available but limited licenses. However the issue at hand is that w&b repo is just a client library, which cannot operate with the closed source server. At this stage the way we can think about this is, if there is closed source code limitations with paywall imposed, it would fall under the commercial section. For the time being we can keep the W&B framework in the commercial section - if you come across other frameworks that have these restrictions then we should put them here.

For the rest of the frameworks like pachyderm with source-available but limited licenses, we can think of adding a notice, but that could be a new github issue where it can be explored separate to the W&B / closed source / paywall discussion.

**Comment by zhimin-z** [comment]
> 

Changed!

**Review by axsaucedo — APPROVED** [review]
Thank you for the contribution @zhimin-z !

**Pull request merged**